content/src/cloud/remote-ui/remote-ui-source-code.md

```diff
@@ -0,0 +1,17 @@
+---
+zendesk:
+  article_id: 26510075061021
+  name: Remote UI - source code on GitHub
+  position: 15
+  labels: cloud
+---
+
+The source code for the Remote UI is available on GitHub:
+
+- [SniTun](https://github.com/NabuCasa/snitun) - End-to-End encryption with SNI proxy on top of a TCP multiplexer
+- [hass-nabucasa](https://github.com/NabuCasa/hass-nabucasa) - Cloud integration in Home Assistant
+
```

Comment:
https://github.com/home-assistant/core/tree/dev/homeassistant/components/cloud The cloud integration is also a vital part of this.

Reply:
thanks for the review, @ludeeus :pray:

Reply:
> https://github.com/home-assistant/core/tree/dev/homeassistant/components/cloud

it seems they are both called Cloud integration?